frontend/src/components/common/table/table.tsx

```diff
@@ -39,6 +47,8 @@ const Table: FC<Props> = ({
   const hasStrPlaceholder = Boolean(placeholder);
   const hasData = data.length !== 0;
   const hasPlaceholder = hasStrPlaceholder && !hasData;
+  const hasPagination = Boolean(pagination);
+  const paginationProps = usePagination(pagination);
```

Comment:
we should call usePagination in only it tables, where we need it. not inside in the common table